openy.info.yml

```diff
@@ -82,7 +82,7 @@ dependencies:
   - openy_system
   - openy_taxonomy
   - openy_tour
-  - openy_upgrade_tool
+  #- openy_upgrade_tool
```

Comment:
why?
Should this wait until https://github.com/ymcatwincities/openy/pull/1466 is merged?

Reply:
fixed